libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.ts

```diff
@@ -94,14 +101,16 @@ export class SkyAgGridCellEditorLookupComponent
     this.#changeDetector.markForCheck();
   }
 
+  public override isAlive = (): boolean => this.#isAlive;
```

Comment:
I modified the editor to extend `PopoverComponent` rather than just the interface. We had a property called `isAlive` but the component uses a method to check `isAlive()` — I don't know if that is a coincidence, but they collide. I've renamed our property.